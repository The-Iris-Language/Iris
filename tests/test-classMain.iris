.. test for declaring class
class Main () {
  int main() {
    return 0;
  }
  
  int x;

  bool univ hotdog() {
    return false;
  }

  int mustard;

  string relish;
}