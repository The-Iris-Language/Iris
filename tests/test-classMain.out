class Main of Object () {                    
int main() {
return 0;
}
int x;
bool univ hotdog() {
return false;
}
int mustard;
string relish;
}